pages/linux/edquota.md

```diff
@@ -0,0 +1,23 @@
+# edquota
+
+> Edit quota for a user or group.
+
+- Edit the user quota (default):
```

Comment:
If equota is run without flags it is run for the current user.  Should I add a line for 'Edit quota of current user', then change line 5 to 'Edit quota for specified user'. ?